tensorrt_llm/_torch/autotuner.py

```
@@ -1170,6 +1260,253 @@ def _profile_runners(
+            f"[Autotuner] Created persistent subprocess pool with "
+            f"{worker_count} workers for device_index={device_index}.")
+        return pool
+
+    def shutdown_subprocess_pools(self) -> None:
+        """Shut down all persistent worker pools. Safe to call repeatedly."""
+        pools, self._subprocess_pools = self._subprocess_pools, {}
+        for pool in pools.values():
+            try:
+                pool.shutdown(wait=True)
+            except Exception:
+                pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)